FreeRTOS/Demo/RISC-V-Qemu-virt_GCC/main.c

```diff
@@ -30,6 +30,10 @@
 
 /* Run a simple demo just prints 'Blink' */
 #define DEMO_BLINKY	1
+#define mainVECTOR_MODE_DIRECT	1
+
+extern void freertos_risc_v_trap_handler( void );
```

Comment:
Can you help me find where is this defined? I cannot seem to find it.

Or is this supposed to be defined by the application as a 'hook'?

Reply:
The interrupt_handler is defined kernel side. You can find it in: `FreeRTOS-Kernel/portable/GCC/riscV/portASM.S`